src/graphicswin.cpp

```diff
@@ -1175,10 +1175,7 @@ void GraphicsWindow::MenuEdit(Command id) {
             }
             // Regenerate, with these points marked as dragged so that they
             // get placed as close as possible to our snap grid.
-            SS.GW.ClearPending();
```

Comment:
Wow! This is code from 11 years ago :-)
https://github.com/solvespace/solvespace/blame/094eff755b035bab2d5eb8037f5264b0033f3ec5/src/graphicswin.cpp#L1178